addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComHandler.java

```diff
@@ -45,14 +44,11 @@
  * @author Pauli Anttila - Initial contribution
  */
 public class RFXComHandler extends BaseThingHandler implements DeviceMessageListener {
+    private final static int LOW_BATTERY_LEVEL = 1;
 
-    private Logger logger = LoggerFactory.getLogger(RFXComHandler.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(RFXComHandler.class);
```

Comment:
No but it was surrounded by `private static final Logger logger` declarations which I changed to `private static final Logger LOGGER`. 

And I (and most companies I worked) actually use and prefer the static ones, what are the/your objections against it.